signac/contrib/project.py

```diff
@@ -1836,6 +1879,30 @@ def get_project(root=None, search=True):
         with a root directory identical to the specified root arugment.
     :type search: bool
     :returns: The project handle.
+    :rtype: :py:class:`~.Project`
     :raises LookupError: If no project configuration can be found.
     """
     return Project.get_project(root=root, search=search)
+
+
+def get_job(root=None):
+    """When the current working directory or provided root directory is a job
+    workspace directory (or subdirectory thereof), locate the closest parent
+    project and return this job.
```

Comment:
Same comment as above, the first paragraph in a multi-line doc string should span only one line.